modules/transport-netty4/src/test/java/org/elasticsearch/http/netty4/Netty4HttpServerTransportTests.java

```diff
@@ -90,12 +90,12 @@ public void shutdown() throws Exception {
     public void testCorsConfig() {
         final Set<String> methods = new HashSet<>(Arrays.asList("get", "options", "post"));
         final Set<String> headers = new HashSet<>(Arrays.asList("Content-Type", "Content-Length"));
-        final String suffix = randomBoolean() ? " " : ""; // sometimes have a leading whitespace between comma delimited elements
+        final String prefix = randomBoolean() ? " " : ""; // sometimes have a leading whitespace between comma delimited elements
```

Comment:
This value is passed as 'prefix' parameter to the `collectionToDelimitedString(coll, delim, prefix, suffix)`.
And since that method was affected by this change I thought it might be a good idea to also eliminate this possible source of confusion.

But sure, I'll undo it tomorrow.